helix-term/src/ui/picker.rs

```diff
@@ -114,7 +125,7 @@ impl<T: Item> FilePicker<T> {
     pub fn new(
         options: Vec<T>,
         editor_data: T::Data,
-        callback_fn: impl Fn(&mut Context, &T, Action) + 'static,
+        callback_fn: impl Fn(&mut Context, Option<&T>, Action) + 'static,
```

Comment:
The None case allows us to represent when the picker is dismissed, otherwise we have to define a separate `on_close` method, you can see the attempt here: https://github.com/helix-editor/helix/pull/5492/commits/34f15552a086a005b3411908f6031c59b3db2d73